mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```
@@ -55,6 +55,12 @@ examples:
     primary_resource_id: 'featureview'
     vars:
       name: 'example_feature_view'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'vertex_ai_featureonlinestore_featureview_with_beta_fields'
```

I'd recommend using a name based on the feature demonstrated rather than the fact the fields happen to be beta currently.

```suggestion
    name: 'vertex_ai_featureonlinestore_featureview_with_vector_search'
```